Documentation/Index.rst

```diff
@@ -121,21 +121,16 @@ all relevant changes for each TYPO3 version since 7.
 
 .. _start-configuration:
 
-Configure TYPO3
-===============
-
-One of the major features of TYPO3 is its configurability. The :ref:`t3coreapi:config-overview`
-in "TYPO3 Explained" gives you an overview of various configuration languages.
+Configure your TYPO3 installation
+=================================
 
-Specifically, you might want to
+TYPO3 is well known for being extensive and easy to configure. The :ref:`t3coreapi:config-overview`
+in `TYPO3 Explained <https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/Index.html>`__ gives you an overview of the various configuration options available to you.
```

Comment:
Can use the `:ref:` approach to `start` entry point for this manual? That would reduce maintenance costs>